.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
```

Comment:
Don't know much about this. If you're sure it doesn't break it, it's fine with me